doc/src/programmers-guide.md

```diff
@@ -14,7 +14,7 @@ This document describes the development workflow when writing AtomVM application
 
 ## AtomVM Features
 
-Currently, AtomVM implements a strict subset of the BEAM instruction set, as of Erlang/OTP R21.  Previous versions of Erlang/OTP are not supported.
+Currently, AtomVM implements a strict subset of the BEAM instruction set, as of Erlang/OTP.  Previous versions of Erlang/OTP are not supported.
```

Comment:
Replace “Previous versions…” with “For information about specific versions of required software, see the [Release Notes](release-notes.md).
”